cucumber.js

```diff
@@ -0,0 +1,11 @@
+let common = [
+  'bdd/features/**/*.feature', // Specify our feature files
```

Comment:
Can we do this way:
- Rename `bdd/` to `tests/` (because it is really not necessary to show what kind of testing technique we are using  - be it BDD or TDD)
- Move `srcs/tests/setup.ts` to new directory `tests/`
- Rename `step_definitions/` folder to just `steps/`, because it is redundant I think
- Rename `ExampleSteps.ts` to just `Example.ts`, because it is redundant too